packages/galeri/src-xpetra/Galeri_Elasticity2DProblem.hpp

```diff
@@ -125,7 +125,7 @@ namespace Galeri {
     template <typename Scalar, typename LocalOrdinal, typename GlobalOrdinal, typename Map, typename Matrix, typename MultiVector>
     Teuchos::RCP<Matrix> Elasticity2DProblem<Scalar,LocalOrdinal,GlobalOrdinal,Map,Matrix,MultiVector>::BuildMatrix() {
       using Teuchos::SerialDenseMatrix;
-      typedef Teuchos::ScalarTraits<SC> TST;
+      typedef Teuchos::ScalarTraits<SC> STS;
```

Comment:
This is not a big deal to me, but why did you change TST to STS?

Reply:
Cause it's that way in Xpetra/MueLu... And I don't recall why.